libraries/chain/include/graphene/chain/protocol/ext.hpp

```diff
@@ -229,4 +229,14 @@ void unpack( Stream& s, graphene::chain::extension<T>& value, uint32_t _max_dept
 
 } // fc::raw
 
+template<typename T> struct get_typename< graphene::chain::extension<T> >
+{ 
+   static const char* name()
+   { 
+      return (std::string("graphene::chain::extension<") 
+         + fc::get_typename<T>::name() + std::string(">")).c_str();   
```

Comment:
Need to change to `static n` here as well?

Reply:
Yes. Done. Thanks.